specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -7909,6 +7909,10 @@
           "$ref": "#/definitions/WebHookActivityTypeProperties"
         }
       },
+      "policy": {
```

Comment:
If you want to add a new property, you should add it under properties.

Reply:
we can't do that as app model doesn't support it.

Below is another commit done in same way for your reference.
https://github.com/Azure/azure-rest-api-specs/commit/a0232d7d2c4c19396f58da0b78ab3c98c27bf415

Reply:
@kalyanrk61 Looks to me like you did not actually do it the same way, because the closing braces don't match up right.